tests/sentry/api/endpoints/test_organization_monitors.py

```diff
@@ -1,79 +1,49 @@
-from exam import fixture
-
 from sentry.models import Monitor, MonitorStatus, MonitorType, ScheduleType
 from sentry.testutils import APITestCase
 
 
-class ListOrganizationMonitorsTest(APITestCase):
-    @fixture
-    def org(self):
-        return self.create_organization(owner=self.user, name="baz")
-
-    @fixture
-    def team(self):
-        return self.create_team(organization=self.org, members=[self.user])
+class OrganizationMonitorsTestBase(APITestCase):
+    endpoint = "sentry-api-0-organization-monitors"
 
-    @fixture
-    def project(self):
-        return self.create_project(teams=[self.team])
+    def setUp(self):
+        super().setUp()
+        self.login_as(self.user)
 
-    @fixture
-    def path(self):
-        return f"/api/0/organizations/{self.org.slug}/monitors/"
 
+class ListOrganizationMonitorsTest(OrganizationMonitorsTestBase):
     def check_valid_response(self, response, expected_monitors):
-        assert response.status_code == 200, response.content
```

Comment:
why wasn't this converted to use `get_success_response`?

Reply:
I converted the `self.client.get`s to use `get_success_response`. The status code check in `check_valid_response` is redundant.